Dockerfile.venv

```diff
@@ -0,0 +1,14 @@
+FROM elifesciences/python:b22fbad8f66100d88cb4bd7c7d092b376a9e09bc
+
+ENV PROJECT_FOLDER=/srv/profiles
+WORKDIR ${PROJECT_FOLDER}
+
+USER root
+RUN chown --recursive elife:elife .
+
+USER elife
+
+RUN virtualenv venv
+
+COPY requirements.txt ./
```

Comment:
There may be other steps not yet implemented, but if all this did was to create `venv/` the only difference with having it in the main `Dockerfile` would be the lack of a `requirements.txt` in the shipped one. Unless the base image of `Dockerfile` change to something smaller.